packages/kbn-optimizer/src/worker/webpack.config.ts

```diff
@@ -188,7 +188,7 @@ export function getWebpackConfig(bundle: Bundle, bundleRefs: BundleRefs, worker:
                         loaderContext,
                         Path.resolve(
                           worker.repoRoot,
-                          `src/core/public/core_app/styles/_globals_${theme}.scss`
+                          `src/core/public/styles/core_app/_globals_${theme}.scss`
```

Comment:
@elastic/kibana-operations since `CoreApp` was moved to packages, I moved the associated styles to `src/core/public/styles/core_app` as it's what we've been doing with other domains, so I had to update the webpack config and the associated integration tests. 

Note that given these styles are effectively only 'used' by the optimizer, we could eventually think about moving them to `@kbn/optimizer`, but I didn't want to have this discussion within the current PR.